### DISCUSSION TITLE
How to implement Prometheus Metrics into Grafana

### DISCUSSION THREAD

**Discussion by whit3str** [Zibbp/ganymede#125] [General]
Hi there,

Kinda lost and kinda new to this, what does /metrics refers to in https://github.com/Zibbp/ganymede/wiki/Prometheus-Metrics ?

**Comment by Zibbp** [comment]
The metric endpoint is exposed at `/metrics` on the API container so if using the default docker-compose it would be `IP:4800/metrics`. You then need to setup a Prometheus scrape job hitting that `/metrics` endpoint.
